### PR TITLE
Translating reference react dom server.md

### PR DESCRIPTION
**Pull request by splimter**

### CODE REVIEW

File: content/docs/reference-react-dom-server.md [hunk 1]
```diff
@@ -17,76 +17,76 @@ var ReactDOMServer = require('react-dom/server');
 
 ## Overview {#overview}
```
Comment (iRayan7) [inline content/docs/reference-react-dom-server.md]: overview -> لمحة عامة

File: content/docs/reference-react-dom-server.md [hunk 2]
```diff
@@ -17,76 +17,76 @@ var ReactDOMServer = require('react-dom/server');
...
+تعتمد هذه التوابع الإضافية على الحزمة (stream) والتي لا تتوفر إلا على الخادم ولا تعمل على المتصفح.
 
-- [`renderToNodeStream()`](#rendertonodestream)
-- [`renderToStaticNodeStream()`](#rendertostaticnodestream)
+- [`()renderToNodeStream`](#rendertonodestream)
+- [`()renderToStaticNodeStream`](#rendertostaticnodestream)
 
 * * *
 
 ## Reference {#reference}
```
Comment (iRayan7) [inline content/docs/reference-react-dom-server.md]: reference -> مرجع

**Comment by splimter** [comment]
Check the new fixes <3

**Comment by iRayan7** [comment]
Hi Merah, Could please make a new pr with only `reference-react-dom-server.md` been edited?
I can't merge the current pr with 2 other files been edited.

**Pull request closed without merging**